Model/GastatsAd.php

```diff
@@ -93,8 +93,8 @@ public function processGAStats($start_date=null, $end_date=null, $refresh=false)
 		}
 
 		//DFP Data
-		$this->processDFPTotals($start_date, $end_date);
-		$this->processDFPReach($start_date, $end_date);
+		//$this->processDFPTotals($start_date, $end_date);
+		//$this->processDFPReach($start_date, $end_date);
```

Comment:
Do we still want this commented?

Reply:
I believe so. When I met with Mark, he mentioned that the DFP functions are not being used and recommended commenting out these calls.